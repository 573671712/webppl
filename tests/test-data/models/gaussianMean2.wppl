var model = function() {
  var m = gaussian(0, 10);
  var genData = repeat(3, function() {return gaussian(m, 0.1)});
  var score = mapReduce1(plus,
                         function(g) {return gaussianERP.score([4.0, 0.04], g)},
                         genData);
  factor(score)
  return m
}
