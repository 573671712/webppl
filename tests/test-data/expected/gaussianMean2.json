{
  "mean": 4.0,
  "std": 0.5
}
